toplevel/workerLoop.ml

```diff
@@ -10,6 +10,7 @@
 
 let rec parse = function
   | "--xml_format=Ppcmds" :: rest -> parse rest
+  | "-batch" :: rest -> parse rest
```

Comment:
Maybe this should be added to `stm/asynctaskqueue` ?

Reply:
done